app/javascript/src/components/Dashboard/Notes/NewNoteForm.jsx

```diff
@@ -31,25 +26,81 @@ export default function NewNoteForm({ onClose, refetch }) {
     >
       {({ isSubmitting }) => (
         <Form>
-          <Input label="Title" name="title" className="mb-6" />
-          <Textarea label="Description" name="description" rows={8} />
+          <Input
+            className=""
+            label="Title"
+            name="title"
+            placeholder="Enter Title"
+            required
+            size="small"
+          />
+          <Input
+            className="pt-6"
+            label="Description"
+            name="description"
+            placeholder="Enter note description"
+            required
+            size="large"
+          />
+          <Select
+            className="pt-6"
+            isClearable
+            isSearchable
+            label="Assigned Contact"
+            name="contact"
+            options={[
+              {
+                label: "Value One",
+                value: "value1"
+              },
+              {
+                label: "Value Two",
+                value: "value2"
+              }
+            ]}
+            placeholder="Select Role"
+            required
+            size="small"
+          />
+          <Select
+            className="pt-6"
+            isMulti
+            label="Tags"
+            name="tags"
+            options={[
+              {
+                label: "Value One",
+                value: "value1"
+              },
+              {
+                label: "Value Two",
+                value: "value2"
+              }
+            ]}
+            placeholder="Select Role"
+            required
+            size="small"
+          />
```

Comment:
Rather than adding `pt-6` or `mb-6` please use ` space-y-*` utils from tailwindCSS on the parent wrapper. Also, we use margins in such cases over padding.